manifests/0000_26_cloud-controller-manager-operator_16_credentialsrequest-gcp.yaml

```diff
@@ -15,10 +15,45 @@ spec:
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: GCPProviderSpec
-    predefinedRoles:
-      - roles/compute.instanceAdmin
-      - roles/compute.loadBalancerAdmin
-      - roles/iam.serviceAccountUser
+    permissions:
+    - "compute.addresses.create"
+    - "compute.addresses.delete"
+    - "compute.addresses.get"
+    - "compute.addresses.list"
+    - "compute.firewalls.create"
+    - "compute.firewalls.delete"
+    - "compute.firewalls.get"
+    - "compute.firewalls.update"
+    - "compute.forwardingRules.create"
+    - "compute.forwardingRules.delete"
+    - "compute.forwardingRules.get"
+    - "compute.healthChecks.create"
+    - "compute.healthChecks.delete"
+    - "compute.healthChecks.get"
+    - "compute.healthChecks.update"
+    - "compute.httpHealthChecks.create"
+    - "compute.httpHealthChecks.delete"
+    - "compute.httpHealthChecks.get"
+    - "compute.httpHealthChecks.update"
+    - "compute.instanceGroups.addInstances"
+    - "compute.instanceGroups.create"
+    - "compute.instanceGroups.delete"
+    - "compute.instanceGroups.get"
+    - "compute.instanceGroups.listInstances"
+    - "compute.instanceGroups.removeInstances"
+    - "compute.instances.delete"
+    - "compute.instances.get"
+    - "compute.regionBackendServices.create"
+    - "compute.regionBackendServices.delete"
+    - "compute.regionBackendServices.get"
+    - "compute.regionBackendServices.update"
+    - "compute.targetPools.addInstance"
+    - "compute.targetPools.create"
+    - "compute.targetPools.delete"
+    - "compute.targetPools.get"
```

Comment:
I would expect `update` and `use` here as well.

Reply:
These are also not included in the list of service in the code.